Fix invalid credential error handling in authentication process (#962)
vmware/terraform-provider-vcd: .changes/v3.9.0/962-bug-fixes.md

> * Fix a bug, that would not return an error while authenticating with invalid password [GH-962]

Reviewer: Actually, I like your other PR's in govcd description better 
https://github.com/vmware/go-vcloud-director/pull/536/files#diff-2cb2af9c1a524452ed1490c48b24cf528ea3e20c1a2785bba2cc6533635f1168R1

```suggestion
* Fix a bug that prevented returning a specific error while authenticating provider with invalid password [GH-962]
```

Author: Changed.
